app/src/main/java/com/example/androidapplicationtask/BindingAdapter.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.androidapplicationtask
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+import com.squareup.picasso.Picasso
+
+
+@BindingAdapter("setBookmarkImage")
+fun ImageView.setBookmarkImage(flag: Int){
+
+
+        if (flag != 1) {
+            this.setImageResource(R.drawable.holo_bookmark)
+        } else {
+            this.setImageResource(R.drawable.solid_bookmark)
+        }
+
+
+}
+@BindingAdapter("setFlashes")
+fun ImageView.setFlashes(flashes : Post){
```

Comment:
method argument should be url not post object

Reply:
done